src/daemon/mod.rs

```diff
@@ -411,6 +411,20 @@ pub(super) async fn start(
             )
             .await
         });
+
+        let finality = chain_config.policy.chain_finality;
+        let chain = config.chain.to_string();
+        services.spawn_blocking(move || {
+            crate::f3::run_f3_sidecar_if_enabled(
+                format!("http://{rpc_address}/rpc/v1"),
+                crate::rpc::f3::get_f3_rpc_endpoint().to_string(),
+                finality,
+                std::env::var("FOREST_F3_DB_PATH")
+                    .unwrap_or_else(|_| format!("/var/tmp/f3-db-{chain}")),
```

Comment:
Why that location and not in the Forest data directory?

Reply:
Fixed.